exec/internal/compile/compile.go

```diff
@@ -154,6 +176,7 @@ func Compile(disassembly []disasm.Instr) ([]byte, []*BranchTable) {
 			instr.Immediates = []interface{}{instr.Immediates[1].(uint32)}
 		case ops.If:
 			curBlockDepth++
+			emitMetadata(OpJmpZ, buffer.Len(), 9)
```

Comment:
the size is almost always `9`.
perhaps add a `const` with a nice name that explains why and what it is?
and use that everywhere?
(there's only one occurence of a size of 18.)